### ISSUE TITLE
Emulation Station update seems to have broken this theme

### ISSUE THREAD

**Issue by gilbertohasnofb** [lilbud/es-theme-minimal#2]
I recently uploaded my Emulation Station and it seems it broke this theme: no images are displayed any longer. All icons in the RetroPie menu are missing, as well as all cover art and game descriptions. If I switch to the default carbon theme I can see all cover art and descriptions again.

**Comment by lilbud** [comment]
I don't have a pi available at the moment (dog ate my sd card.) So I'm not really sure why the update would break everything.

Could you send me your es_log.txt? I'll take a look.

**Comment by gilbertohasnofb** [comment]
Sure, here is my log:

```
lvl2: 	EmulationStation - v2.8.0rp, built Feb  9 2019 - 22:55:44
lvl2: 	Parsing XML file "/home/pi/.emulationstation/resources/mamenames.xml"...
lvl2: 	Parsing XML file "/home/pi/.emulationstation/resources/mamebioses.xml"...
lvl2: 	Parsing XML file "/home/pi/.emulationstation/resources/mamedevices.xml"...
lvl2: 	Creating surface...
lvl2: 	Created window successfully.
lvl2: 	Added known joystick Microsoft X-Box 360 pad (instance ID: 0, device index: 0)
lvl2: 	Checking available OpenGL extensions...
lvl2: 	 ARB_texture_non_power_of_two: MISSING
lvl2: 	Loading system config file /etc/emulationstation/es_systems.cfg...
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "amstradcpc" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "arcade" has no games! Ignoring it.
lvl2: 	Parsing XML file "/home/pi/RetroPie/roms/atari2600/gamelist.xml"...
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "atari5200" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "atari7800" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "atari800" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "atarilynx" has no games! Ignoring it.
lvl2: 	Parsing XML file "/home/pi/.emulationstation/gamelists/c64/gamelist.xml"...
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "fba" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "fds" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "gamegear" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "gb" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "gba" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "gbc" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "mame-libretro" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "mame-mame4all" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "mastersystem" has no games! Ignoring it.
lvl2: 	Parsing XML file "/home/pi/RetroPie/roms/megadrive/gamelist.xml"...
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl2: 	Parsing XML file "/home/pi/RetroPie/roms/n64/gamelist.xml"...
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "neogeo" has no games! Ignoring it.
lvl2: 	Parsing XML file "/home/pi/RetroPie/roms/nes/gamelist.xml"...
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "./../art/${system.theme}.svg" (which resolved to "/etc/emulationstation/themes/minimal/layouts/../art/ngp.svg") 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "ngp" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "ngpc" has no games! Ignoring it.
lvl2: 	Parsing XML file "/home/pi/.emulationstation/gamelists/pc/gamelist.xml"...
lvl1: 	File "/home/pi/RetroPie/roms/pc/DOSBOX_DOOM/DOOM.EXE" does not exist! Ignoring.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "pcengine" has no games! Ignoring it.
lvl2: 	Parsing XML file "/home/pi/RetroPie/roms/psx/gamelist.xml"...
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl2: 	Parsing XML file "/home/pi/.emulationstation/gamelists/retropie/gamelist.xml"...
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "sega32x" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "segacd" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "sg-1000" has no games! Ignoring it.
lvl2: 	Parsing XML file "/home/pi/RetroPie/roms/snes/gamelist.xml"...
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "vectrex" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	System "zxspectrum" has no games! Ignoring it.
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	NinePatchComponent missing texture!
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl1: 	  Warning from theme "/etc/emulationstation/themes/minimal/theme.xml"
  (from included file "/etc/emulationstation/themes/minimal/layouts/V2.xml")
    could not find file "" 
lvl2: 	 req sound [basic.launch]
lvl2: 	   (missing)
lvl2: 	Attempting to launch game...
lvl2: 		sudo /home/pi/RetroPie-Setup/retropie_packages.sh retropiemenu launch /home/pi/RetroPie/retropiemenu/esthemes.rp </dev/tty >/dev/tty
lvl2: 	Creating surface...
lvl2: 	Created window successfully.
lvl2: 	Added known joystick Microsoft X-Box 360 pad (instance ID: 0, device index: 0)
lvl2: 	Added known joystick Microsoft X-Box 360 pad (instance ID: 0, device index: 0)
lvl2: 	Added/Updated 1 entities in '/home/pi/.emulationstation/gamelists/retropie/gamelist.xml'
lvl2: 	EmulationStation cleanly shutting down.
```

**Comment by gilbertohasnofb** [comment]
I see that [this recent commit](https://github.com/RetroPie/EmulationStation/commit/9af271d36f119717dfa2eca41a22509cc237682c) changed some of the logic involving the box art, could it be what caused the issue?

edit: by the way, I am happy to help out testing things on my Pi.

**Comment by lilbud** [comment]
I went over the the Retropie discord to see if anyone there knows what the problem could be. Someone suggested setting your gamelist view to 'detailed'. That might fix it.

I'll be without a Pi to test until next week. It'll take that long for the new sd card to ship. I have another one, but it is wired into my arcade cabinet and without network, so I can't test anything on that.

**Comment by gilbertohasnofb** [comment]
@lilbud I did a fresh reinstall and the problem seem to have gone away. I had experienced a problem with one of the latests commits of Emulation Station, which broke my system (and apparently of a bunch of other people too). I believe that though I thought I had fixed it there were still problems, and this could have been one of the side effects. 

TLDR; reinstalled RetroPie, all is fine now. Many thanks for the support.
